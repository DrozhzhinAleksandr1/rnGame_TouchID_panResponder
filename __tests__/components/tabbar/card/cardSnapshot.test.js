import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Card } from '../../../../app/components/tabbar/card/card';
import { localization } from '../../../../app/common/localization';

const mockItem = {'ID': 3, 'market': 'Currency', 'titleQuotation': 'AUD/CAD', 'priceBuy': "40.298", 'priceSell': "991.03"};

it('create snapshot Card', () => {
    const renderer = new ShallowRenderer()
    let cardScreenSnapshot = renderer.render(<Card item={mockItem} key={mockItem.ID} currentLanguage={localization.ru}/>);
    expect(cardScreenSnapshot).toMatchSnapshot();
});