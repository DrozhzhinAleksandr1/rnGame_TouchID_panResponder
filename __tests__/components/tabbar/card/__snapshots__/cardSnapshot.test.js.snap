// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Card 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#131F2A",
      "elevation": 2,
      "marginBottom": 4,
      "marginLeft": 8,
      "marginRight": 4,
      "shadowColor": "rgba(0, 0, 0, 0.2)",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowRadius": 2,
    }
  }
>
  <TouchableWithoutFeedback
    onPress={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1D2E3E",
        "flex": 1,
        "flexDirection": "row",
        "height": 72,
        "justifyContent": "space-around",
        "marginTop": 4,
        "shadowColor": "rgba(0, 0, 0, 0.2)",
        "shadowOffset": Object {
          "height": 6,
          "width": 0,
        },
        "shadowOpacity": 1,
        "shadowRadius": 6,
        "width": "100%",
        "zIndex": 1,
      }
    }
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#1D2E3E",
          "flex": 1,
          "flexDirection": "row",
          "height": 72,
          "justifyContent": "space-around",
          "marginTop": 4,
          "shadowColor": "rgba(0, 0, 0, 0.2)",
          "shadowOffset": Object {
            "height": 6,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 6,
          "width": "100%",
          "zIndex": 1,
        }
      }
    >
      <Component
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "height": "100%",
            "justifyContent": "center",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "Roboto-Regular",
              "fontSize": 15,
              "height": 16,
              "lineHeight": 16,
            }
          }
        />
      </Component>
      <Component
        style={
          Object {
            "flex": 2,
            "height": "100%",
          }
        }
      >
        <Component
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "height": "100%",
              "marginBottom": 8,
              "marginTop": 8,
            }
          }
        >
          <CardQuotation />
          <Component
            style={
              Object {
                "borderColor": "#5A6772",
                "borderRightWidth": 1,
                "height": "100%",
              }
            }
          >
            <Component
              style={
                Object {
                  "borderColor": "#5A6772",
                  "borderRightWidth": 1,
                  "height": "100%",
                }
              }
            />
          </Component>
          <CardQuotation />
        </Component>
      </Component>
    </Component>
  </TouchableWithoutFeedback>
</Component>
`;
